Update SPM instructions (#15)
transcend-io/consent-manager-ios-sample-sdk: README.md

### Step 1: Include Our Framework

- You can integrate the Framework into your application using one of the following methods:

> ### Manual Steps

Reviewer: nit: maybe for the methods use four `#` instead of three since they are methods nested under Step 1? not super important tho

Author: Updated the readme with few more changes.

    - `pod 'Transcend'`
- Run `pod install` to fetch the Transcend Framework.

### Using Swift Package Manager

```
    https://github.com/transcend-io/Transcend-spm-sdk.git
```

## Usage
Please consult the documentation [here](https://docs.transcend.io/docs/consent-management/mobile-consent/ios) for more details.

### Initialization of API instance 

> The referrence for API instance in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/sampleSDKApp.swift#L45)

Reviewer 2: ```suggestion
The reference for the API instance in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/sampleSDKApp.swift#L45)
```
sp

> - Note: This will not render Consent Banner on UI

Reviewer 2: could you add more detail here? im not really sure what this means

Author: Ahh!! Added few more details. Its basically, the instance that allows you to talk to airgap.js api which is hidden behind all the views. User calls `getRemies` or `getConsent` before showing the UI instance

```
    import SwiftUI
    import Transcend
    
    // Usage
    // completionHandler
    let didFinishNavigation: ((Result<Void, Error>) -> Void) = { result in
      switch result {
        case .success():
          // Your logic goes here
        case .failure(let error):
          print("Error during web view navigation: \(error.localizedDescription)")
      }
    }
    // Create TranscendCoreConfig
    let simpleCoreConfig: TranscendCoreConfig = TranscendCoreConfig(

>       transcendConsentUrl: "https://transcend-cdn.com/cm/{Bundle_Id}/airgap.js", mobileAppId: "com.example.ios")

Reviewer 2: can we maybe pull Bundle_Id and mobileAppId both out as constants and add them to the top of the file? i think we tend to do that for various ids in our code snippets (see [here](https://docs.transcend.io/docs/consent-management/configuration/loading-asynchronously#2.-create-a-script-element))

    
    TranscendWebViewUI(transcendCoreConfig: simpleCoreConfig, didFinishNavigation: didFinishNavigation)
```

### API Usage

> The referrence for API Usage in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/HomeView.swift#L33).

Reviewer 2: ```suggestion
The reference for API Usage in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/HomeView.swift#L33).
```
sp

- Full list of support APIs are listed [here](https://docs.transcend.io/docs/consent-management/mobile-consent/ios/api#1.0.9:definitions-and-usage-of-the-api).
```
// Usage
// Note: can be used only after didFinishNavigation returns .success
TranscendWebView.transcendAPI.webAppInterface.getRegimes(completionHandler: { result, error in
  if let error = error {
    // Your logic goes here
    print("UI Error : \(error)")
  } else {
    // Your logic goes here
    if result?.contains("us") == true {
      self.showTranscendWebView = true
    }
  }
})
```

### Show Consent banner

> The referrence to show consent banner in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/HomeView.swift#L121)

Reviewer 2: ```suggestion
A reference for showing the consent banner in this repository can be found [here](https://github.com/transcend-io/consent-manager-ios-sample-sdk/blob/dev/sampleSDK/HomeView.swift#L121)
```

```
import Transcend

struct ContentView: View {
    let simpleCoreConfig: TranscendCoreConfig = TranscendCoreConfig(

>       transcendConsentUrl: "https://transcend-cdn.com/cm/{Bundle_Id}/airgap.js", mobileAppId: "com.example.ios")

Reviewer 2: same comment about putting constants at the top

    - `pod 'Transcend'`
- Run `pod install` to fetch the Transcend Framework.

### Using Swift Package Manager

```
    https://github.com/transcend-io/Transcend-spm-sdk.git
```

> 

Reviewer: not super familiar with the swift package manager. are there any other instructions on how to use it in the mentioned url?

Author: They just specify the git url on their` Package.swift` file which would download the resources from https://github.com/transcend-io/Transcend-spm-sdk.git.

Reviewer: Thanks for explaining! Maybe add a sentence like that above the url just in case?
